src/main/java/com/amazon/opendistroforelasticsearch/security/support/ConfigConstants.java

```diff
@@ -98,10 +98,12 @@
 
     public static final String OPENDISTRO_SECURITY_USER = OPENDISTRO_SECURITY_CONFIG_PREFIX+"user";
     public static final String OPENDISTRO_SECURITY_USER_HEADER = OPENDISTRO_SECURITY_CONFIG_PREFIX+"user_header";
+    public static final String OPENDISTRO_SECURITY_USER_STRING = OPENDISTRO_SECURITY_CONFIG_PREFIX + "user_string";
```

Comment:
Is it used?

Reply:
Removed. Will be part of separate CR if user_roles string can't be reused